client/app/utils/i18n.js

```diff
@@ -0,0 +1,7 @@
+function translate(translations) {
```

Comment:
I'd change the name of this file to translationUtils.js or i18nUtils.js. Without path, I'd assume this would contain something completely different (like something from http://i18njs.com)